index.html

```diff
@@ -142,6 +142,19 @@ <h3 id="svg11-aria">ARIA in SVG 1.1</h3>
           }
         </script>
       </section>
+      <section>
+        <h3 id="elements-html">The Elements of HTML</h3>
+        <p>
+          Defines the semantics of the Elements of HTML and the web developer rules (author conformance requirements) for the use of the Elements of HTML in HTML 5.1. It also defines requirements for Conformance Checking tools.
+        </p>
+        <script type="application/webspec+json">
+          {
```

Comment:
I mean this part. It is different JSON from the one used in the rest of the document. I didn't mean the HTML spec part :)